tox.ini

```diff
@@ -58,16 +58,16 @@ deps =
     oldestdeps: zeep<3.5.0
     oldestdeps: h5netcdf<0.9
     oldestdeps: cdflib<0.3.20
-    oldestdeps: pytest<5.5
-    oldestdeps: pytest-xdist<1.28
+    oldestdeps: pytest<7.0
+    oldestdeps: pytest-xdist<2.1
     # These are specific extras we use to run in specific factors
     online: pytest-rerunfailures
     online: pytest-timeout
     online: astroquery
     hypothesis: astroquery
     # Figure tests need a tightly controlled environment
     figure-!devdeps: astropy==4.2.1
-    figure-!devdeps: matplotlib==3.3.2
+    figure-!devdeps: matplotlib==3.5.2
```

Comment:
Now this has been bumped, does the `.json` file at https://github.com/sunpy/sunpy/tree/main/sunpy/tests need renaming for the new version?

Reply:
My bad, I see this has been done